src/components/SlateEditor/plugins/comment/CommentForm.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Form, Formik } from "formik";
+import { useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import { Descendant } from "slate";
+import styled from "@emotion/styled";
+import { ButtonV2 } from "@ndla/button";
+import { colors, misc, spacing } from "@ndla/core";
+import { FieldErrorMessage, Label } from "@ndla/forms";
+import { CommentEmbedData } from "@ndla/types-embed";
+import { editorValueToPlainText, plainTextToEditorValue } from "../../../../util/articleContentConverter";
+import { FormControl, FormField } from "../../../FormField";
+import validateFormik, { RulesType } from "../../../formikValidationSchema";
+import PlainTextEditor from "../../PlainTextEditor";
+
+const StyledPlainTextEditor = styled(PlainTextEditor)`
```

Comment:
Hvorfor trenger dette å være en PlainTextEditor, egentlig? Kan det ikke bare være et TextArea?

Reply:
Det ble gjort sånn for å slippe å oversette mellom slate og plaintekst